scripts/xenadmin-build.sh

```diff
@@ -207,7 +212,7 @@ cd ${REPO}/XenAdmin/TestResources && zip -r ${OUTPUT_DIR}/${BRANDING_BRAND_CONSO
 cd ${REPO}/CFUValidator/bin/Release && zip ${OUTPUT_DIR}/CFUValidator.zip ./{*.dll,CFUValidator.exe,${BRANDING_BRAND_CONSOLE_NO_SPACE}Main.exe}
 
 #now package the pdbs
-cp ${REPO}/packages/*.pdb ${OUTPUT_DIR}
+[[ -e ${REPO}/packages/*.pdb ]] && cp ${REPO}/packages/*.pdb ${OUTPUT_DIR}
```

Comment:
Do we want this check here? I'd think we want it to fail if there are no pdbs so we know something went wrong.

Reply:
It was failing for me, but the rest of the installation went ahead just fine. The pdbs for the packages were present in the Debug/Release subfolders in bin, but not in `packages`.

If that's a configuration mistake of my VS2019, I can remove this

Reply:
Ah, just run the download_packages.ps1 adding the switch `-SYMBOLS`. (`Get-Help .\download_packages.ps1` shows the available switches).

Reply:
ok sounds good. I'll test it and if it works I'll remove the existence check 👍